server.js

```diff
@@ -1,21 +1,43 @@
 require("dotenv").config();
 /* ==== External Modules ==== */
 const express = require("express");
-const methodOverride = require("method-override");
+const bodyParser = require("body-parser");
+const logger = require('morgan');
+const router = express.Router();
 
 /* ==== Internal Modules ==== */
-const routes = require("./routes");
-
+const MongoClient = require('mongodb').MongoClient
 /* ==== Instanced Modules  ==== */
 const app = express();
 
+
+// Routes
+const routes = require("./routes");
+require('./config/database');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7821591)
```suggestion
require("./config/database");
```